### PR TITLE
test: add changes required for fedora 30 AMI

### PR DESCRIPTION
**Pull request by haircommander**
Including removing our own debugging callback
change from package to a manual shell call (because python2-dnf is sunsetted, but we still need python2 for oct)
disable nodev for userns test
Signed-off-by: Peter Hunt pehunt@redhat.com

### CODE REVIEW

**Comment by haircommander** [comment]
I manually tagged the fedora ami as ready, despite it failing at integration test step. things may break in other branches FYI, I'll revert before EOD if I can't prove the other branches aren't affected

**Comment by haircommander** [comment]
**NOTE: I set the new ami to ready, meaning it is running f30. This may affect other branches**

**Comment by mrunalp** [comment]
```
Unexpected error:
    <exec.CodeExitError>: {
        Err: {
            s: "error running &{/go/src/k8s.io/kubernetes/_output/bin/kubectl [kubectl --server=https://172.18.31.25:6443 --kubeconfig=/var/run/kubernetes/admin.kubeconfig exec hostpath-injector --namespace=volume-2678 -- /bin/sh -c echo 'Hello from csi-hostpath from namespace volume-2678' > /opt/0] []  <nil>  /bin/sh: can't create /opt/0: Permission denied\ncommand terminated with exit code 1\n [] <nil> 0xc001c944e0 exit status 1 <nil> <nil> true [0xc002c12c80 0xc002c12c98 0xc002c12cb0] [0xc002c12c80 0xc002c12c98 0xc002c12cb0] [0xc002c12c90 0xc002c12ca8] [0x10f0c10 0x10f0c10] 0xc002022c00 <nil>}:\nCommand stdout:\n\nstderr:\n/bin/sh: can't create /opt/0: Permission denied\ncommand terminated with exit code 1\n\nerror:\nexit status 1",
        },
        Code: 1,
    }
    error running &{/go/src/k8s.io/kubernetes/_output/bin/kubectl [kubectl --server=https://172.18.31.25:6443 --kubeconfig=/var/run/kubernetes/admin.kubeconfig exec hostpath-injector --namespace=volume-2678 -- /bin/sh -c echo 'Hello from csi-hostpath from namespace volume-2678' > /opt/0] []  <nil>  /bin/sh: can't create /opt/0: Permission denied
```
This doesn't look like a flake.

**Comment by haircommander** [comment]
seems you're right:
https://openshift-gce-devel.appspot.com/build/origin-federated-results/pr-logs/pull/cri-o_cri-o/2876/test_pull_request_crio_e2e_fedora/7537/ (from 1.16)

**Comment by haircommander** [comment]
(I reproduced on a VM ,and reverted the AMI changes so other branches can move forward)

**Comment by haircommander** [comment]
/hold cancel

we will need this to merge PRs now that we switched to f30
/test e2e_fedora

File: contrib/test/integration/ansible.cfg [hunk 1]
```diff
@@ -58,7 +58,7 @@ gather_subset = network
 host_key_checking = False
 
 # change the default callback
-#stdout_callback = skippy
+stdout_callback = debug
```
Comment (saschagrunert) [inline contrib/test/integration/ansible.cfg]: Is this still wanted? If yes what does it do? :innocent:

Comment (haircommander) [reply contrib/test/integration/ansible.cfg]: before, we had our own stdout callback (the removed callback_plugins python code). It caused  a cyclical import problem, and now ansible ships its own stdout callback for debugging, which this line adds :)

Comment (saschagrunert) [reply contrib/test/integration/ansible.cfg]: Alright, thanks. :+1:

**Comment by haircommander** [comment]
carrying https://github.com/cri-o/cri-o/pull/2893 here too, so manually merge when e2e_fedora passes

**Comment by saschagrunert** [comment]
I think this is ready for the manual merge, right?

**Comment by haircommander** [comment]
Yeah this is ready!

**Comment by saschagrunert** [comment]
> Yeah this is ready!

Closing my eyes and pressing the button :see_no_evil:

**Pull request merged**